client/src/components/pages/application/Signup/index.js

```diff
@@ -1,5 +1,156 @@
 import React from 'react';
+import axios from 'axios';
+import Card from '../../../common/card';
+import Typography from '../../../common/Typography';
+import TextField from '../../../common/TextField';
+import Button from '../../../common/Button';
+import signupValidate from '../../../../utils/application/SignupValidation';
+import SelectBox from '../../../application/SelectBox';
+import './index.css';
 
-const SignUp = () => <h1>SignUp</h1>;
+import signupImage from '../../../../assets/images/signup.svg';
+
+const initialState = {
+  fullName: '',
+  email: '',
+  password: '',
+  passwordConfirmation: '',
+  location: '',
+  message: '',
+};
+class SignUp extends React.Component {
+  state = initialState;
+
+  handleSignIn = () => {
+    const { history } = this.props;
+    history.push('/login');
+  };
+
+  throwMessage = (message) => this.setState({ message });
+
+  resetForm = () => {
+    this.setState(initialState);
+  };
+
+  submit = () => {
+    const { message, ...lest } = this.state;
+    signupValidate(lest).then((result) =>
```

Comment:
after posting correct data. its better to display a success message as notification with green background for example  to show user that his registration going well .